api/src/test/java/uk/gov/dhsc/htbhf/claimant/converter/EligibilityOverrideToEligibilityOverrideDTOConverterTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.dhsc.htbhf.claimant.converter;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.dhsc.htbhf.claimant.model.EligibilityOverrideDTO;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static uk.gov.dhsc.htbhf.TestConstants.MAGGIE_AND_LISA_DOBS;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.EligibilityOverrideTestDataFactory.aConfirmedEligibilityOverrideWithChildren;
+
+class EligibilityOverrideToEligibilityOverrideDTOConverterTest {
+
+    private EligibilityOverrideToEligibilityOverrideDTOConverter converter = new EligibilityOverrideToEligibilityOverrideDTOConverter();
+
+    @Test
+    void shouldConvertEligibilityOverrideToEligibilityOverrideDTO() {
+        //Given
+        var eligibilityOverride = aConfirmedEligibilityOverrideWithChildren(MAGGIE_AND_LISA_DOBS);
```

Comment:
As much as I love using var, at the beginning of the project the team voted not to use it unless it's with a new statement. However, we're leaving the project so I guess it's up to you and @adeel718  to decide on the policy going forward.

Reply:
Can you change this for now and I will check with other projects around the policy.

Reply:
done

Reply:
Yeah, I'm not a big fan of var, I really don't think adds much at all apart from occasional confusion over the type being used.